test/unit/services/personal/update-personal-phone-numbers-change-service.test.js

```diff
@@ -0,0 +1,83 @@
+// Test framework dependencies
+import { describe, test, expect, beforeEach, vi } from 'vitest'
+
+// Things we need to mock
+import { fetchPersonalChangeService } from '../../../../src/services/personal/fetch-personal-change-service.js'
+import { flashNotification } from '../../../../src/utils/notifications/flash-notification.js'
+import { updateDalService } from '../../../../src/services/DAL/update-dal-service.js'
+import { updatePersonalPhoneNumbersMutation } from '../../../../src/dal/mutations/personal/update-personal-phone-numbers.js'
+
+// Test helpers
+import { mappedData } from '../../../mocks/mock-personal-details.js'
+
+// Thing under test
+import { updatePersonalPhoneNumbersChangeService } from '../../../../src/services/personal/update-personal-phone-numbers-change-service.js'
+
+// Mocks
+vi.mock('../../../../src/services/personal/fetch-personal-change-service.js', () => ({
+  fetchPersonalChangeService: vi.fn()
+}))
+
+vi.mock('../../../../src/utils/notifications/flash-notification.js', () => ({
+  flashNotification: vi.fn()
+}))
+
+vi.mock('../../../../src/services/DAL/update-dal-service.js', () => ({
+  updateDalService: vi.fn().mockResolvedValue({})
+}))
+
+describe('updatePersonalPhoneNumbersChangeService', () => {
+  let yar
+  let credentials
+
+  beforeEach(() => {
+    vi.clearAllMocks()
+
+    mappedData.changePersonalPhoneNumbers = {
+      businessTelephone: null,
```

Comment:
What's being set here is businessTelephone and businessMobile. Makes me think the test isn't testing properly since these are incorrect, and it's still passing